### ISSUE TITLE
MongoDB ^3.3.0 not shimmed.

### ISSUE THREAD

**Issue by lenartbezek** [elastic/apm-agent-nodejs#1456]
**Describe the bug**

MongoDB is not shimmed for versions `^3.3.0`.

**To Reproduce**

**script.js**
```js
require("elastic-apm-node").start({
    logLevel: "debug"
});

require("express");
require("mongodb");
```

**package.json**
```json
{
  "name": "test-mongodb-shim",
  "version": "1.0.0",
  "main": "index.js",
  "dependencies": {
    "elastic-apm-node": "^3.0.0",
    "express": "^4.17.1",
    "mongodb": "^3.3.0"
  }
}
```

**output**
```sh
> node ./script.js
adding hook to Node.js module loader
shimming finalhandler@1.1.2 module
shimming http@12.8.0 module
shimming http.Server.prototype.emit function
shimming http.ServerResponse.prototype.writeHead function
shimming http.request function
shimming http.get function
shimming express@4.17.1 module
shimming express.Router.use function
shimming express.static function
copying property mime from express.static
no active transaction found - cannot build new span
intercepted call to http.request { id: null }
```

**expected output** (on mongodb version `3.2.7`)
```
> node .\index.js
adding hook to Node.js module loader
shimming finalhandler@1.1.2 module
shimming http@12.8.0 module
shimming http.Server.prototype.emit function
shimming http.ServerResponse.prototype.writeHead function
shimming http.request function
shimming http.get function
shimming express@4.17.1 module
shimming express.Router.use function
shimming express.static function
copying property mime from express.static
shimming mongodb-core@3.2.7 module
shimming mongodb-core.Server.prototype.command
shimming mongodb-core.Server.prototype functions: [ 'insert', 'update', 'remove', 'auth' ]    
shimming mongodb-core.Cursor.prototype functions: [ 'next', '_getmore' ]
no active transaction found - cannot build new span
intercepted call to http.request { id: null }
```

**Comment by watson** [comment]
MongoDB 3.3.0 changed the way it handles queries internally, so the latest version of the agent doesn't, unfortunately, support that or newer versions. However, we already landed support for these versions in `master` and they will be part of the next release (see #1423).

**Comment by Qard** [comment]
Should be fixed in 3.1.0, which I've released now.
